src/app.ts

```diff
@@ -0,0 +1,22 @@
+import express, { Application, Request, Response, NextFunction } from 'express';
+const app: Application = express();
+const cors = require('cors');
+import postRoutes from './routes/posts' // bring in the routes to use here and then remove them below
+
+app.set('port', process.env.PORT || 5000);
```

Comment:
With TS you can use [nullish coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing). I.e, `process.env.PORT ?? 5000` over the `||` operator. Generally would recommend it over `||`